migrations/20200606105847-convert-collective-long-description-to-html.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+import Promise from 'bluebird';
+import showdown from 'showdown';
+
+const converter = new showdown.Converter();
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const collectives = await queryInterface.sequelize.query(
+      `
+        SELECT * FROM "Collectives"
+        WHERE LENGTH("longDescription") > 0 AND "longDescription" NOT LIKE '<%';
+      `,
+      { type: Sequelize.QueryTypes.SELECT },
+    );
+
+    await Promise.map(collectives, collective =>
+      queryInterface.sequelize.query(
+        `
+        UPDATE "Collectives" c
+        SET "longDescription" = :longDescription
+        WHERE c.id = :id
+      `,
+        {
+          replacements: {
+            longDescription: converter.makeHtml(collective.longDescription),
+            id: collective.id,
+          },
+        },
+      ),
+    );
+  },
+
+  down: async (queryInterface, Sequelize) => {
+    const collectives = await queryInterface.sequelize.query(
+      `
+        SELECT * FROM "Collectives"
+        WHERE LENGTH("longDescription") > 0;
+      `,
+      { type: Sequelize.QueryTypes.SELECT },
+    );
+
+    await Promise.map(collectives, collective =>
+      queryInterface.sequelize.query(
+        `
+        UPDATE "Collectives" c
+        SET "longDescription" = :longDescription
+        WHERE c.id = :id
+      `,
+        {
+          replacements: {
+            longDescription: converter.makeMarkdown(collective.longDescription),
+            id: collective.id,
+          },
+        },
+      ),
+    );
```

Comment:
The purpose of `down` is to rollback the changes if something wrong happens during the migration. What that would mean here is that we would need something to convert from HTML to markdown. We don't have that, and we probably don't want to introduce more complexity just to solve it. So doing this is fine:

```suggestion
    // Can't rollback this one
```